src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1021,7 +1021,7 @@ impl DdlController {
                                             upstream_actor_id: vec![],
                                             upstream_fragment_id,
                                             upstream_dispatcher_type: PbDispatcherType::Hash as _,
-                                            fields: sink_fields.to_vec(),
+                                            fields: vec![],
```

Comment:
Discussed offline. @Chiro11 mentions that the `fields` won't be used. Will have a PR later to mark this field as deprecated.